environment_cpu.yml

```
@@ -0,0 +1,34 @@
+name: graph
+channels:
+  - pytorch
+  - pyg
+  - conda-forge
+  - defaults
+dependencies:
+  - numcodecs
+  - pandas
+  - pip
+  - pyg
+  - python=3.12
+  - pytorch cpuonly -c pytorch
```

This should need to be this, you don't need to add the channel in the environment

```suggestion
  - pytorch
  - cpuonly
```